xml/System.Net.Http/HttpMessageHandler.xml

```diff
@@ -196,11 +196,12 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="request">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="request">The HTTP request message to send.</param>
+        <param name="cancellationToken">The cancellation token to cancel operation.</param>
+        <summary>When overridden in a derived class, sends an HTTP request. Otherwise, throws a <see cref="T:System.NotImplementedException" />.</summary>
```

Comment:
```suggestion
        <summary>When overridden in a derived class, sends an HTTP request with the specified request and cancellation token. Otherwise, throws a <see cref="T:System.NotImplementedException" />.</summary>
```